payment-stellar/src/main/java/org/stellar/anchor/paymentservice/stellar/requests/SubmitTransactionRequest.java

```diff
@@ -0,0 +1,10 @@
+package org.stellar.anchor.paymentservice.stellar.requests;
+
+import lombok.Data;
+
+@Data
+public class SubmitTransactionRequest {
+  String envelope;
+
+  public SubmitTransactionRequest(String envelope) {}
```

Comment:
missing `this.envelope = envelop`
If this is a simple class, we may consider doing
```
@AllArgsConstructor
@Data
public class SubmitTransactionRequest {
  String envelope;
}
```